pip-requirements.txt

```diff
@@ -12,7 +12,7 @@
 # https://www.python.org/dev/peps/pep-0440/#version-specifiers
 ##
 
-edk2-pytool-library~=0.11.0 # MU_CHANGE - update to 0.11.0 or later
-edk2-pytool-extensions~=0.17.0 # MU_CHANGE - update to 0.15.0 or later
+edk2-pytool-library~=0.11.2 # MU_CHANGE - update to 0.11.0 or later
+edk2-pytool-extensions~=0.17.1 # MU_CHANGE - update to 0.15.0 or later

```

Comment:
Maybe update the comments to match the newer versions.  I don't believe everything has been backwards compatible.

Reply:
Updated.